interface/patient_file/report/custom_report.php

```diff
@@ -188,27 +188,6 @@ function zip_content($source, $destination, $content = '', $create = true)
       // by div of navigateLink class. Specifically used for CAMOS, but
       // can also be used by other forms that require output in the
       // encounter listings output, but not in the custom report. ?>
```

Comment:
page is cut off at top

![image](https://user-images.githubusercontent.com/278968/91674748-cb2c1000-eaee-11ea-93d9-f5b4901135f4.png)

Reply:
the Find feature is working, but pretty sure it is supposed to highlight the term. In this case it focuses on Sad at bottom, but is not highlighting the terms.

![image](https://user-images.githubusercontent.com/278968/91674792-f7e02780-eaee-11ea-907b-853d33ebdcfe.png)

Reply:
How do I to trigger the search? In my environment, the select didn't have any options

Reply:
Just need to select the forms from the patient report (I basically just do a Check All). Then they will show up. And the search only then searches the text within the forms that are selected (they are all selected by default).

Reply:
Doy you mean, when I searched the test, the result will highlight the search value.

Like this?
![image](https://user-images.githubusercontent.com/22230889/91703301-8bc3f900-ebac-11ea-8238-5394053991c1.png)

Reply:
Check out how it work in current codebase. In the case if this PR, I think likely removed the styling that is used to highlight the items (the functionality of the search is working, it just lost the highlighting).

Reply:
Ok, I tried to test it on the OpenEMR demo version, the highlighting feature is working. So I will get in-depth to fix it.

Reply:
hi @stu01509 ,
Just tried most recent code in this PR. Getting close Couple more issues:
1. The top part needs to stay visible when scrolling down
2. Need to ensure highlighting stays visible when click Prev and Next

Reply:
Hi @bradymiller 

Update the commit, I adjust the CSS priority and dynamic to get the search bar element height that the search bar can fixed on top.